### ISSUE TITLE
App-Crash when opening live stream

### ISSUE THREAD

**Issue by cemrich** [cemrich/zapp#138]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void de.christinecoenen.code.zapp.app.player.BackgroundPlayerService$a.a(android.content.Intent)' on a null object reference
at de.christinecoenen.code.zapp.app.livestream.ui.detail.ChannelDetailActivity.m(ChannelDetailActivity.java:337)
at de.christinecoenen.code.zapp.app.livestream.ui.detail.ChannelDetailActivity.lambda$6qn4GMJb1dQ9ELO-RxA-2UOqP6U(ChannelDetailActivity.java)
at de.christinecoenen.code.zapp.app.livestream.ui.detail.-$$Lambda$ChannelDetailActivity$6qn4GMJb1dQ9ELO-RxA-2UOqP6U.run(lambda)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5258)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:963)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:758)
```

I can't reproduce this bug right now but it occurs on a variety of Android versions and devices.

**Android versions:** 5.1.1 - 9
**App versions:** 2.0.0 and 2.1.0

**Comment by cemrich** [comment]
It looks like OnItemSelected is called before the player service has been bound.

**Comment by cemrich** [comment]
I can reproduce this bug on the emulator when I set ```activity_channel_detail_play_stream_delay_millis``` to ```0```. It seems to be caused by a timing issue during binding / starting playback.

**Comment by cemrich** [comment]
This issue should be fixed in v2.1.1which is hitting F-Droid the next few days.
